app/utils/press-release-snapshot.js

```diff
@@ -1,39 +1,68 @@
 import { tracked } from '@glimmer/tracking';
 
+function serializePublicationChannels(publicationChannels) {
+  let publicationChannelsStringArray = [];
+  let publicationChannelsString = '';
```

Comment:
Waarom deze initializatie?

Reply:
om alles niet te kort te maken, maar blijkbaar wordt het dan te lang, ik zal in de toekomst hierop letter